daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/StablePackages.scala

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.lf.language
+
+import com.daml.lf.data.Ref.{PackageId, PackageName}
+
+object StablePackages {
+  // Based on compiler/damlc/tests/src/stable-packages.sh
+  // TODO: Use this map to generate the mapping used in the test,
+  // or generate both from a single source.
+  val map = Map[PackageName, PackageId](
```

Comment:
We don't need to expose this.
```suggestion
  private val nameToIdMapping = Map[PackageName, PackageId](
```